### DISCUSSION TITLE
Kong

### DISCUSSION THREAD

**Discussion by ocbrollingpaper** [supabase/supabase#5815] [Feature Requests]
Hey guys, this project is amazing.
I don't think I should create issue so I opened discussion.
Me and my company are enterprise users of Kong, I was wondering will you support Kong enterprise?

Would you accept PR for guide how to upgrade Kong to Enterprise?

**Comment by FJEagle** [comment]
It seems "plan to use Edge Runtime as a replacement for Kong, acting as an API gateway to other Supabase services.", according to https://supabase.com/blog/edge-runtime-self-hosted-deno-functions
